WooCommerce/Classes/POS/Controllers/PointOfSaleOrderController.swift

```diff
@@ -123,7 +130,11 @@ final class PointOfSaleOrderController: PointOfSaleOrderControllerProtocol {
                                                  order: updatedOrder,
                                                  giftCard: nil,
                                                  fields: fieldsToUpdate,
-                                                 onCompletion: { result in
+                                                 onCompletion: { [weak self] result in
+                guard let self = self else { return }
+                if case .success = result {
+                    self.celebrate()
```

Comment:
I didn't know we had a celebration protocol 😀

I don't have strong objections but for me the sound is a UI/UX related, similarly to animation or confetti effects, therefore it's more appropriate for them to be placed closely to the view. Either in view itself or within the model. What do you think?

Reply:
> I didn't know we had a celebration protocol

It has some funny history behind.  When we ran unit tests locally you suddenly got hit with ~10 "chaching" sounds full-blast coming from your machine totally unexpectedly which wasn't pleasant 😂 Now this is disabled for dev builds

>  I don't have strong objections but for me the sound is a UI/UX related, similarly to animation or confetti effects, therefore it's more appropriate for them to be placed closely to the view. Either in view itself or within the model. What do you think

That makes sense, and sounds good, I guess even in the view/viewhelper itself it's fine to call the class and method directly as it's very encapsulated 🤔  I'll play a bit more with the tests before merging anything.